e2e/regressions/build.go

```diff
@@ -65,5 +97,7 @@ func RunE2ETests(env e2e.TestEnv) func(*testing.T) {
 	return func(t *testing.T) {
 		// https://github.com/sylabs/singularity/issues/4203
 		t.Run("Issue4203", c.issue4203)
+		// https://github.com/sylabs/singularity/issues/4517
+		t.Run("Issue4517", c.issue4517)
```

Comment:
it's not a problem to make the first argument to t.Run readable: `t.Run("issue 4517", c.issue4517)`

I believe master looks more like the suggestion right now.